packages/platform-sdk-ada/src/services/transaction.ts

```diff
@@ -44,26 +47,41 @@ export class TransactionService implements Contracts.TransactionService {
 		);
 
 		// Get a `Bip32PrivateKey` instance according to `CIP1852` and turn it into a `PrivateKey` instance
-		const privateKey = getCip1852Account(
-			input.sign.mnemonic,
-			this.#config.get<number>("network.crypto.slip44"),
-		).to_raw_key();
+		const accountKey: Bip32PrivateKey = deriveAccountKey(deriveRootKey(input.sign.mnemonic), 0);
+		const publicKey = accountKey.to_public();
 
-		// These are the inputs (UTXO) that will be consumed to satisfy the outputs. Any change will be transfered back to the sender
-		const utxos: any = await this.listUnspentTransactions(input.from);
+		/* TODO: We need to determine how to specify the input.
```

Comment:
@faustbrian Need your input here.

Reply:
I think we would need to use the extended public key here which we should add as a new input property so the consumer can explicitly state that they want the package to perform some kind of derivation to gather all data necessary.